packages/bsky/tests/views/labels-needs-review.test.ts

```diff
@@ -43,127 +43,272 @@ describe('bsky needs-review labels', () => {
     await network.processAll()
 
     AtpAgent.configure({ appLabelers: [network.ozone.ctx.cfg.service.did] })
-    await network.bsky.db.db
-      .insertInto('label')
-      .values({
-        src: network.ozone.ctx.cfg.service.did,
-        uri: sc.dids.geoff,
-        cid: '',
-        val: 'needs-review',
-        exp: null,
-        neg: false,
-        cts: new Date().toISOString(),
-      })
-      .execute()
   })
 
   afterAll(async () => {
     await network.close()
   })
 
-  it('applies to thread replies.', async () => {
-    const {
-      data: { thread },
-    } = await agent.app.bsky.feed.getPostThread({
-      uri: sc.posts[sc.dids.alice][0].ref.uriStr,
+  describe('account-level', () => {
+    beforeAll(async () => {
+      await network.bsky.db.db
+        .insertInto('label')
+        .values({
+          src: network.ozone.ctx.cfg.service.did,
+          uri: sc.dids.geoff,
+          cid: '',
+          val: 'needs-review',
+          exp: null,
+          neg: false,
+          cts: new Date().toISOString(),
+        })
+        .execute()
     })
-    assert(isThreadViewPost(thread))
-    expect(
-      thread.replies?.some((reply) => {
-        return (
-          isThreadViewPost(reply) && reply.post.author.did === sc.dids.geoff
-        )
-      }),
-    ).toBe(false)
-  })
 
-  it('applies to quote lists.', async () => {
-    const {
-      data: { posts },
-    } = await agent.app.bsky.feed.getQuotes({
-      uri: sc.posts[sc.dids.alice][0].ref.uriStr,
+    afterAll(async () => {
+      await network.bsky.db.db
+        .deleteFrom('label')
+        .where('src', '=', network.ozone.ctx.cfg.service.did)
+        .execute()
     })
-    expect(
-      posts.some((post) => {
-        return post.author.did === sc.dids.geoff
-      }),
-    ).toBe(false)
-  })
 
-  it('applies to reply, quote, and mention notifications.', async () => {
-    const {
-      data: { notifications },
-    } = await agent.app.bsky.notification.listNotifications(
-      {},
-      {
-        headers: await network.serviceHeaders(
-          sc.dids.alice,
-          ids.AppBskyNotificationListNotifications,
-        ),
-      },
-    )
-    expect(
-      notifications.some((notif) => {
-        return notif.reason === 'reply' && notif.author.did === sc.dids.geoff
-      }),
-    ).toBe(false)
-    expect(
-      notifications.some((notif) => {
-        return notif.reason === 'quote' && notif.author.did === sc.dids.geoff
-      }),
-    ).toBe(false)
-    expect(
-      notifications.some((notif) => {
-        return notif.reason === 'mention' && notif.author.did === sc.dids.geoff
-      }),
-    ).toBe(false)
-  })
+    it('applies to thread replies.', async () => {
```

Comment:
I recommend reviewing this file with whitespace ignored.  The same tests are essentially run twice, first with geoff labeled, and then all of geoff's posts labeled.